docs/jitserver_tuning.md

```diff
@@ -120,6 +120,8 @@ You can enable the provision of performance metrics by specifying the `-XX:+JITS
 
 :fontawesome-solid-pencil-alt:{: .note aria-hidden="true"} **Note:** There is a limit of four concurrent `GET` requests at any given time.
 
+You can use the [`-XX:JITServerMetricsSSLKey`](xxjitservermetricssslkey.md) and [`-XX:JITServerMetricsSSLCert`](xxjitservermetricssslkey.md) options for encrypting the data with SSL or TLS.
```

Comment:
I would use here "SSL/TLS".  When I see "SSL or TLS" makes me think that I have a choice. OpenSSL will pick the most secure option that can be negotiated between the client and the server. SSL is deprecated and TLS is everywhere now, so for the vast majority of the cases TLS is going to be used.

Reply:
As per the IBM style guidelines, slashes are considered ambiguous and should be avoided. Since slash in this instance means 'or', it has been replaced so.